src/utils.ts

```diff
@@ -918,3 +918,23 @@ export function readSecretValue(prompt: string, dataFile?: string): Promise<stri
     throw e;
   }
 }
+/**
+ * Updates or creates a .gitignore file with the given entries in the given path
+ */
+export function updateOrCreateGitignore(dirPath: string, entries: string[]) {
+  const gitignorePath = path.join(dirPath, ".gitignore");
+
+  if (fs.existsSync(gitignorePath)) {
+    let content = fs.readFileSync(gitignorePath, "utf-8");
+    for (const entry of entries) {
+      if (!content.includes(entry)) {
+        content += `${entry}\n`;
```

Comment:
Should this be `\n${entry}` to account for .gitignores without a new line at the end of the file?

Reply:
uhm, `${entry}\n` should still work because we're using `.includes` above instead of `===`

Reply:
Sorry I'm not following. I was saying we may want to append `\n${entry}` in case content was something like the following: `line1\nline2`. 

Appending `${entry}\n` would result in the following: `line1\nline2${entry}\n` which would be problematic